src/Microsoft.AspNet.Authentication.OAuth/OAuthHandler.cs

```diff
@@ -174,10 +113,12 @@ public OAuthHandler(HttpClient backchannel)
             requestMessage.Headers.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
             requestMessage.Content = requestContent;
             var response = await Backchannel.SendAsync(requestMessage, Context.RequestAborted);
-            response.EnsureSuccessStatusCode();
-            var payload = JObject.Parse(await response.Content.ReadAsStringAsync());
-
-            return new OAuthTokenResponse(payload);
+            if (response.IsSuccessStatusCode)
```

Comment:
My main point is we didn't throw before (we turned them into no-op successful no tickets) and we had test coverage for these scenarios, so unless we want to turn all of them into 500s, this was closest to the old behavior (at least for tests).  Basically I replaced try/catch return empty result, with return null instead of throw, and Fail